ang/crmCaseType.js

```diff
@@ -145,7 +154,16 @@
       'sequence': 'Sequence'
     };
 
-    $scope.caseType = apiCalls.caseType ? apiCalls.caseType : _.cloneDeep(newCaseTypeTemplate);
+    if (apiCalls.caseType) {
```

Comment:
Is `apiCalls` a variable name you are obliged to use? If not, then I am not sure this is an accurate enough variable name. It is simply `data`, isn't it?

Reply:
`apiCalls` is a collection of API calls that is resolved before loading the `CaseTypeCtrl` controller. You can see the definition here: https://github.com/compucorp/civicrm-core/blob/ab71b8c6bb9990e72d7179e58d3b67520dce46ef/ang/crmCaseType.js#L50

I don't like the name either since yes, it begins as API calls, but ends upd as API call results, but that's the convention used in this and other files.